gosearch.go

```
@@ -159,134 +401,155 @@ func UnmarshalJSON() (Data, error) {
-			WriteToFile(username, fmt.Sprintf("::    Antiviruses: %s\n", v))
-		case []interface{}:
-			antiviruses := make([]string, len(v))
-
-			for i, av := range v {
-				antiviruses[i] = fmt.Sprint(av)
-			}
-
-			avs := strings.Join(antiviruses, ", ")
-			WriteToFile(username, fmt.Sprintf("::    Antiviruses: %s\n", avs))
+		// Add to terminal table
+		table.Append([]string{
```

golangci-lint (1.64.8)

Error return value of `table.Append` is not checked

(errcheck)